app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+namespace App\Http\Controllers;
+use Illuminate\Http\Request;
+use App\User;
```

Comment:
You did not use User here so no need to add.

Reply:
Fixes issue(Delete)